templates/components.html.njk

```diff
@@ -0,0 +1,34 @@
+{% macro speakerPortrait(speaker) %}
+
+  <div class="c-speaker-portrait c-speaker-portrait--variant-{{ speaker.variant }}">
+    {# TODO Use responsive image endcomment #}
+    <img
+      class="c-speaker-portrait__image"
+      src="{{ speaker.portrait }}"
+      alt="Photo of {{ speaker.name }}"
+    />
+  </div>
+
+{% endmacro %}
+
+{% macro speakerGallery(speakers) %}
+
+  <ul class="c-speaker-gallery list pl0">
+    {% for speaker in speakers %}
+      <li class="c-speaker-gallery__item">
+        <a href="#">
```

Comment:
I believe here we should link to the speaker details page? If so, we need one more property in the JSON data.

Reply:
I think we’ll probably have a speaker id and generate the link from that?

Reply:
Let’s re-factor this once we have the spreadsheet import in.